pkg/innerring/config.go

```diff
@@ -194,7 +194,7 @@ func validateBlockchainConfig(cfg *config.Config) error {
 // auto-deployment.
 func setNetworkSettingsDefaults(netCfg *deploy.NetworkConfiguration) {
 	netCfg.MaxObjectSize = 64 << 20 // in bytes of object payload
-	netCfg.EpochDuration = 240      // in FS chain blocks (e.g. ~1h for 15s block interval)
+	netCfg.EpochDuration = 3600     // in seconds
```

Comment:
Please don't do this now. This will delay deployments in some cases, it'll be fixed with initial forced ticks, but this requires a refactor. Also tests are using 1s blocks with this default epoch config and can experience delays.

Reply:
We will change it when we're ready everywhere, that's it.